hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/CompactStreamSerializerTest.java

```diff
@@ -32,13 +32,7 @@
 import com.hazelcast.test.HazelcastParallelClassRunner;
 import com.hazelcast.test.annotation.ParallelJVMTest;
 import com.hazelcast.test.annotation.QuickTest;
-import example.serialization.BitsDTO;
-import example.serialization.EmployeeDTO;
-import example.serialization.EmployeeDTOSerializer;
-import example.serialization.EmployeeWithSerializerDTO;
-import example.serialization.EmployerDTO;
-import example.serialization.ExternalizableEmployeeDTO;
-import example.serialization.NodeDTO;
+import example.serialization.*;
```

Comment:
Let's revert star import to make checkstyle happy

Reply:
Done